pages/santa.js

```diff
@@ -187,7 +187,14 @@ function Signup() {
   const [values, setValues] = useState({})
   return (
     <Base method="get" target="_blank" action="https://hack.af/santa-signup">
-      <Heading sx={{ color: 'black' }}>Register!</Heading>
+      <Heading sx={{ color: 'black', textAlign: 'left' }}>Register!</Heading>
```

Comment:
```suggestion
      <Heading sx={{ color: 'black', textAlign: 'left', mb: 2 }}>Register!</Heading>
```

Could we add a slight margin here?